examples/resolver/resolver.go

```diff
@@ -0,0 +1,28 @@
+package resolver
+
+import (
+	"fmt"
+	"net/http"
+)
+
+//RequestURL simply returns
+//the request url from REQUEST_URI header
+//this should not be done in production applications
+type RequestURL struct {
```

Comment:
Hm, I guess it can be done like in your examples, to just take the base url from the request. Or take the base url out of another provided header that is configurable like `X-Forwarded-For` or sth.